lib/index.js

```diff
@@ -29,6 +29,7 @@ class JsonEditor {
         options.stringify_width = options.stringify_width || 2
         options.stringify_fn = options.stringify_fn || null
         options.stringify_eol = options.stringify_eol || false
+	options.ignoreDots = options.ignoreDots || false;
```

Comment:
Can you please use underscore notation, to be consistent with the other options? Thanks!